go.mod

```diff
@@ -9,7 +9,7 @@ replace github.com/denisenkom/go-mssqldb => github.com/grafana/go-mssqldb v0.0.0
 
 // Override k8s.io/client-go outdated dependency, which is an indirect dependency of grafana/loki.
 // It's also present on grafana/loki's go.mod so we'll need till it gets updated.
-replace k8s.io/client-go => k8s.io/client-go v0.22.1
+replace k8s.io/client-go => k8s.io/client-go v0.25.0
```

Comment:
@gabor -- is this update OK?  loki is now using v25

Reply:
@ryantxu hmm.. actually, i'm not sure if we use loki-code in grafana anymore. when https://github.com/grafana/grafana/commit/da74dba7c8d3da59d4187ae9414c7257985f9e48 was committed, we switched to the streaming-json-parser, and stopped using the loki version. 

(but i'm not sure what that lines does exactly though)